vagrant/values.yaml

```diff
@@ -23,3 +23,6 @@ sumologic:
   accessId: dummy
   accessKey: dummy
   endpoint: http://receiver-mock.receiver-mock:3000/terraform/api/
+telegraf:
+  enabled: true
+  replicaCount: 1
```

Comment:
If we have more than 1 replica do things actually work? Is there a need, perhaps cuz of load, to have more than 1?

Reply:
Yes they will work. Didn't tested the telegraf-operator in terms of performance, but will change the replicaCount to 3 (as it's default value)

Reply:
I am fine with 1. In my experience, there is almost always 1 operator.  I just wanted to understand the risks if customers scale.  eg will it work.